examples/light-switch-app/nrfconnect/README.md

```diff
@@ -577,7 +583,35 @@ To perform binding process you need to complete following steps:
     __{"fabricIndex": 1, "node": <1>, "endpoint": 1, "cluster": 8}__ is a binding for __LevelControl__ cluster. 
 
 
-To test communication between Light Switch and bound device use [buttons](#buttons) or [CLI commands](#matter-cli-commands). Both of them are described in [Device UI section](#device-ui).
+### Binding multicast devices using chip-tool for Windows/Linux
+Binding multicast allows to add all lighting devices to one multicast group. After that Light Switch can send multicast request and all of devices listening bound multicast group can run received command. It can be used to control more than one lighting device via single light switch at the same time.
+
+ITo perform binding for group multicast you need to complete following steps:
+
+1. Navigate to the CHIP root directory:
+
+2. Build chip-tool using [chip-tool guide](../../../docs/guides/chip_tool_guide.md#building).
+
+3. Go to chip-tool build directory.
+
+4. Add ACL (Access Control List) in the lighting endpoint permissions to receive commands from Light Switch:
+
+        chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 3, "authMode": 2, "subjects": [2], "targets": [{"cluster": 6, "endpoint": 1, "deviceType": null}, {"cluster": 8, "endpoint": 1, "deviceType": null}]}]' 1 0
+
+5. Add Light Switch device to multicast group:
```

Comment:
I would not assume here that light switch device has node ID equal to 1, as user may select other value during commissioning.